code/graphics/post_processing.cpp

```diff
@@ -34,6 +35,31 @@ PostEffectUniformType mapUniformNameToType(const SCP_string& uniform_name)
 	}
 }
 
+
+// used by In-Game Options menu
+bool Post_processing_enable_lightshafts = true;
+
+auto LightshaftsOption =
+	options::OptionBuilder<bool>("Graphics.Lightshafts", "Lightshafts", "Enable lightshafts (requires post-processing)")
+		.category("Graphics")
+		.default_val(true)
+		.level(options::ExpertLevel::Advanced)
+		.bind_to(&Post_processing_enable_lightshafts)
+		.importance(60)
+		.finish();
+
+int Post_processing_bloom_intensity = 25; // using default value of Cmdline_bloom_intensity
+
+auto BloomIntensityOption = options::OptionBuilder<int>("Graphics.BloomIntensity",
+	"Bloom intensity",
+	"Set bloom intensity (requires post-processing)")
+	.category("Graphics")
+	.range(0, 200)
```

Comment:
The [FS Wiki says](https://wiki.hard-light.net/index.php/Command-Line_Reference#-bloom_intensity) the range for `-bloom_intensity` is 0-200, but the call site for `gr_bloom_intensity()` in render_options.cpp makes me wonder if it's really 0-100.